test/framework/dump.go

```diff
@@ -209,6 +210,37 @@ func (f *GardenerFramework) dumpGardenerExtension(extension v1alpha1.Object) {
 	}
 }
 
+func (f *CommonFramework) DumpLogsForPodsWithLabelsInNamespace(ctx context.Context, ctxIdentifier string, k8sClient kubernetes.Interface, namespace string, labels client.MatchingLabels) error {
+	pods := &corev1.PodList{}
+	if err := k8sClient.Client().List(ctx, pods, client.InNamespace(namespace), labels); err != nil {
+		return err
+	}
+
+	var result error
+	for _, pod := range pods.Items {
+		if err := f.DumpLogsForPodInNamespace(ctx, ctxIdentifier, k8sClient, namespace, pod.Name); err != nil {
+			result = multierror.Append(result, err)
+		}
+	}
+	return result
+}
+
+func (f *CommonFramework) DumpLogsForPodInNamespace(ctx context.Context, ctxIdentifier string, k8sClient kubernetes.Interface, podNamespace, podName string) error {
+	f.Logger.Infof("%s [NAMESPACE %s] [POD %s] [LOGS]", ctxIdentifier, podNamespace, podName)
+	request := k8sClient.Kubernetes().CoreV1().Pods(podNamespace).GetLogs(podName, &corev1.PodLogOptions{})
```

Comment:
There is already `pkg/client/kubernetes.GetPodLogs` func. We can use it, right?

Reply:
Right, thanks for pointing out.